web/api/v1/api.go

```diff
@@ -574,6 +585,120 @@ func (api *API) alertmanagers(r *http.Request) (interface{}, *apiError, func())
 	return ams, nil, nil
 }
 
+// AlertDiscovery has info for all active alerts.
+type AlertDiscovery struct {
+	ActiveAlerts []*Alert `json:"activeAlerts"`
```

Comment:
Happy to rename this back to `Alerts`. `ActiveAlerts` gives the user a hint, that resolved alerts are not included.